libc/test/src/unistd/fpathconf_test.cpp

```diff
@@ -10,6 +10,7 @@
 #include "hdr/limits_macros.h"
 #include "hdr/sys_stat_macros.h"
 #include "hdr/unistd_macros.h"
+#include "llvm-libc-macros/sys-stat-macros.h"
```

Comment:
Do you have to add this to pass the compilation?

Reply:
Yes, I remember not having it will cause CI error. Do you wanna double check?

Reply:
Also, can you help me merge when you see everything is ok? thanks.

Reply:
You already have `hdr/sys_stat_macros.h` so you should not include `llvm-libc-macros/sys-stat-macros.h`.
Then you should also remove `libc.include.llvm-libc-macros.sys-stat-macros`.